impeller/renderer/backend/vulkan/swapchain/surface_vk.cc

```diff
@@ -24,17 +24,8 @@ std::unique_ptr<SurfaceVK> SurfaceVK::WrapSwapchainImage(
   }
 
   const auto enable_msaa = transients->IsMSAAEnabled();
-
   const auto swapchain_tex_desc = swapchain_image->GetTextureDescriptor();
 
```

Comment:
This texture descriptor was unused.

Reply:
Ah, the descriptor comes from the texture source.